Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -252,46 +252,23 @@ final class UIViewExtensionsTests: XCTestCase {
         waitForExpectations(timeout: 0.5)
     }
 
-    func testRotateByAngle() {
+    func testRotateBy() {
         let view1 = UIView()
         let transform1 = CGAffineTransform(rotationAngle: 2)
-        view1.rotate(byAngle: 2, ofType: .radians, animated: false, duration: 0, completion: nil)
+        view1.rotate(by: .init(value: 2, unit: .radians), animated: false, duration: 0, completion: nil)
         XCTAssertEqual(view1.transform, transform1)
 
         let view2 = UIView()
         let transform2 = CGAffineTransform(rotationAngle: .pi * 90.0 / 180.0)
-        view2.rotate(byAngle: 90, ofType: .degrees, animated: false, duration: 0, completion: nil)
+        view2.rotate(by: .init(value: 90, unit: .degrees), animated: false, duration: 0, completion: nil)
```

Comment:
Same here.
```suggestion
        view2.rotate(by: .degrees(90), animated: false, duration: 0, completion: nil)
```

Reply:
Cool, that's the final format I want for these PRs.

Reply:
That's great, but you still haven't used it here 😛

Reply:
Oh, my bad🙈